src/liiteri/file_cleaner.clj

```diff
@@ -0,0 +1,42 @@
+(ns liiteri.file-cleaner
+  (:require [chime :as c]
+            [clojure.core.async :as a]
+            [clojure.java.jdbc :as jdbc]
+            [clj-time.core :as t]
+            [clj-time.periodic :as p]
+            [com.stuartsierra.component :as component]
+            [liiteri.db.file-metadata-store :as metadata-store]
+            [liiteri.files.file-store :as file-store]
+            [org.httpkit.client :as http]
+            [ring.util.http-response :as response]
+            [taoensso.timbre :as log]))
+
+(defn- clean-files [db storage-engine config]
+  (log/info "Cleaning files")
+  (jdbc/with-db-transaction [datasource db]
+    (doseq [file (metadata-store/get-old-draft-files db)]
+      (log/info (str "Cleaning file: " (:key file)))
+      (file-store/delete-file (:key file) storage-engine db))))
```

Comment:
Does metadata get updated? Or does the file come up as deleted on the next round as well?